### PR TITLE
[Docs][Plus] Add Docs with Sylius Plus deployment

### PR DESCRIPTION
**Pull request by AdamKasp**
| Q               | A
| --------------- | -----
| Branch?         | 1.7
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| License         | MIT

### CODE REVIEW

File: docs/cookbook/deployment/platform-sh.rst [hunk 1]
```diff
@@ -203,6 +203,62 @@ When you get connected please run:
     By default platform.sh creates only one instance of the database with the ``main`` name.
     Platform.sh works with the concept of an environment per branch if activated. The idea is to mimic production settings per each branch.
 
+.. rst-class:: plus-doc
+
+Sylius Plus Deployment
```
Comment (CoderMaggie) [inline docs/cookbook/deployment/platform-sh.rst:208]:
```suggestion
How to deploy Sylius Plus to Platform.sh?
```

File: docs/cookbook/deployment/platform-sh.rst [hunk 2]
```diff
@@ -203,6 +203,62 @@ When you get connected please run:
...
+----------------------
+
+`Sylius Plus <https://sylius.com/plus/>`_ is installed to Sylius like a plugin, but it needs some changes in configuration to deploy it properly.
```
Comment (CoderMaggie) [inline docs/cookbook/deployment/platform-sh.rst:211]:
```suggestion
`Sylius Plus <https://sylius.com/plus/>`_ is installed to Sylius like a plugin, but it needs some changes 
to the Platform.sh configuration presented above to deploy it properly.
```

File: docs/cookbook/deployment/platform-sh.rst [hunk 3]
```diff
@@ -203,6 +203,62 @@ When you get connected please run:
...
+First of all you need configure your project follow by `Sylius Plus installation </book/installation/sylius_plus_installation>`_
```
Comment (CoderMaggie) [inline docs/cookbook/deployment/platform-sh.rst:212]:
```suggestion
First of all, make sure you have your project configured following the `Sylius Plus installation guide. </book/installation/sylius_plus_installation>`_
```

File: docs/cookbook/deployment/platform-sh.rst [hunk 4]
```diff
@@ -203,6 +203,62 @@ When you get connected please run:
...
+After that you can modify your the ``.platform.app.yaml``
```
Comment (CoderMaggie) [inline docs/cookbook/deployment/platform-sh.rst:213]:
```suggestion
After that, you should modify your the ``.platform.app.yaml``
```

Comment (CoderMaggie) [reply docs/cookbook/deployment/platform-sh.rst:213]: I'd mention which step numbers you need to alternate from the original Platform.sh deployment guide you have above.

File: docs/cookbook/deployment/platform-sh.rst [hunk 5]
```diff
@@ -203,6 +203,62 @@ When you get connected please run:
...
+    dependencies:
+        nodejs:
+            yarn: "*"
+            gulp-cli: "*"
+        ruby:
+            "wkhtmltopdf-binary": "> 0.12.5.1" # adding wkhtmltopdf as a one of dependencies
+
+    # ...
+
+For using wkhtmltopdf on server properly, you also need modify ``config\packages'knp_snappy.yaml``:
```
Comment (CoderMaggie) [inline docs/cookbook/deployment/platform-sh.rst:245]:
```suggestion
In order to use the wkhtmltopdf (needed for Invoicing and Refunds) on server properly, you also need to add it to the ``config\packages'knp_snappy.yaml``:
```

File: docs/cookbook/deployment/platform-sh.rst [hunk 6]
```diff
@@ -203,6 +203,62 @@ When you get connected please run:
...
+        enabled:    true
+        binary:     wkhtmltopdf # for local purpose was '%env(WKHTMLTOPDF_PATH)%'
+        options:    []
+    image:
+        enabled:    true
+        binary:     wkhtmltoimage # for local purpose was '%env(WKHTMLTOIMAGE_PATH)%'
+        options:    []
+
+
+Rest of platform.sh configurations are same as with in deployment Sylius
```
Comment (CoderMaggie) [inline docs/cookbook/deployment/platform-sh.rst:260]:
```suggestion
All the other steps from the Sylius deployment on Platform.sh remain unchanged.
```

Please start using dots at the end of sentences or ☠️  😂

File: docs/cookbook/deployment/platform-sh.rst [hunk 7]
```diff
@@ -203,6 +203,61 @@ When you get connected please run:
...
     Platform.sh works with the concept of an environment per branch if activated. The idea is to mimic production settings per each branch.
 
+.. rst-class:: plus-doc
+
+How to deploy Sylius Plus to Platform.sh?
+-----------------------------------------
+
+`Sylius Plus <https://sylius.com/plus/>`_ is installed to Sylius like a plugin, but it needs some changes to the Platform.sh configuration presented above to deploy it properly.
+First of all, make sure you have your project configured following the `Sylius Plus installation guide </book/installation/sylius_plus_installation>`_.
+After that, you should modify your the ``.platform.app.yaml``, configuration from step 2 should be extended by following lines.
```
Comment (CoderMaggie) [inline docs/cookbook/deployment/platform-sh.rst:213]:
```suggestion
After that, you should modify your ``.platform.app.yaml``. Configuration from step 2 should be extended by the following lines.
```

**Comment by GSadee** [comment]
Thanks, Adam! :tada:

**Pull request merged**